backend/src/app/features/components_v2.clj

```diff
@@ -255,6 +255,7 @@
                       (assoc shape
                              :type :frame                  ; Old groups must be converted
                              :fills (or (:fills shape) []) ; to frames and conform to spec
+                             :shapes (or (:shapes shape) [])
```

Comment:
this adds `:shapes` attrs to all shapes

Reply:
No. Only to shapes that are instance head (either they have a `:component-id` attribute or are the root shape inside a v1 component.